src/components/common/MobileContainer.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback, useState } from 'react'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { useMobileResize } from '../../integration/mobile'
+import { Navbar } from './Layout/Navbar'
+import { isEmail, subscribe } from '../../integration/mail'
+import './MobileContainer.css'
+
+export default React.memo(() => {
+  useMobileResize()
+  const [state, setState] = useState({ sending: false, sent: false, error: false as false | Error })
```

Comment:
As error is an instance of `Error`, a non existent error should be considered null.
```suggestion
  const [state, setState] = useState({ sending: false, sent: false, error: null | Error })
```